scripts/bridge/populate_external_bridge.py

```
@@ -0,0 +1,135 @@
+                    ROW_NUMBER() OVER (ORDER BY player_id) as external_player_id,
+                    0 as retrosheet_player_id
+                FROM raw_bref.batting_stats
+                WHERE player_id IS NOT NULL
+                ON CONFLICT (external_source, external_player_id) DO NOTHING
+            """)
+            
+            bref_count = cur.rowcount
+            conn.commit()
+            print(f"Populated {bref_count} Baseball Reference players in bridge.external_player_xref")
+            return bref_count
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)